### DISCUSSION TITLE
Issue with Browse deeplink

### DISCUSSION THREAD

**Discussion by G2Patrik** [phantom/docs#75] [Help & Support]
### Summary

Hi, there seems to be an issue with the browse link when we try to open the URL from our mobile app. If the user did not have the phantom app installed they got redirected to the App/Play store but at the moment we are getting Page not found on Phantom Wallets website. Happens on both iOS and Android.

https://docs.phantom.app/phantom-deeplinks/other-methods/browse#url-structure - from these docs the url we are trying to open looks fine. Here is an example:

https://phantom.app/ul/browse/https%3A%2F%2Fmatrica.io%2Foauth2%3F{example_data_here}?ref=g2esports://connectAccount

Ideally we would want the user to be taken either to the App/Play store or they should get redirected to a download page (https://phantom.app/download). Any help would be much appreciated.

### Steps to Reproduce

1. Create URL (https://phantom.app/ul/browse/${example_url}?ref=g2esports://connectAccount`
2. Open URL using Linking.openURL(url)
3. Get Page not Found

### Phantom Version

Latest

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Thank you for writing in!

It does appear that the url you provided should route to a phantom download resource when phantom is not installed. We are investigating the issue. I will follow up here once I have more information.

**Reply by johnsonchau-bulb** [reply]
@adamdelphantom I can confirm that I am also having the same issue:
https://phantom.app/ul/browse/https%3A%2F%2Fwww.bulbapp.io?ref=https%3A%2F%2Fwww.bulbapp.io

This universal link has worked in the past and follows this documentation: https://docs.phantom.app/phantom-deeplinks/other-methods/browse

**Reply by adamdelphantom** [reply]
Thanks for confirming you're seeing the same thing @johnsonchau-bulb!

I'll update the thread here, once we have a resolution.
